prisma/schema.prisma

```diff
@@ -52,12 +52,12 @@ model subscription_topics {
 }
 
 model subscriptions {
-  id             BigInt    @id @default(autoincrement())
+  id             BigInt   @id @default(autoincrement())
   subscriberId   String
   subscriberType String
   topic          String?
-  createdAt      DateTime? @default(now()) @db.Timestamptz(6)
-  updatedAt      DateTime? @default(now()) @db.Timestamptz(6)
+  createdAt      DateTime @default(now()) @db.Timestamptz(6)
+  updatedAt      DateTime @default(now()) @db.Timestamptz(6)
   type           String
-  notifiedAt     DateTime  @default(now()) @db.Timestamptz(6)
+  notifiedAt     DateTime @default(now()) @db.Timestamptz(6)
```

Comment:
I think this is not a good idea, because it would mean that the first notification the user receives contains every single activity in our database that matches the subscription. Having it mandatory makes sure they get only notified about activities added after they subscribed. At least that made sense to me when I thought about it.